bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -1752,7 +1777,7 @@ spec:
                 env:
                 - name: DISABLE_DSC_CONFIG
                   value: "true"
-                image: REPLACE_IMAGE:latest
+                image: quay.io/edgarhz/odh-operator:latest
```

Comment:
last one to go before merge:  we need revert this line

Reply:
Ouch! I just reverted it.